e2e/Makefile

```diff
@@ -138,9 +146,14 @@ stop-lvmd:
 	$(MAKE) shutdown-kind
 	for i in $$(seq 3); do \
 		if systemctl is-active -q lvmd$$i.service; then $(SUDO) systemctl stop lvmd$$i.service; fi; \
-		for j in $$(seq 3); do \
+		for j in $$(seq 4); do \
 			if [ -f $(BACKING_STORE)/backing_store$${i}_$${j} ]; then \
```

Comment:
This if statement is no longer needed because all nodes have the same backing store files and VGs.

Reply:
Good point! Removed it.